### PR TITLE
adding initial files

### PR DESCRIPTION
**Pull request by strawberrie68**

### CODE REVIEW

File: backend/models/bookModel.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    }, 
+    bookImg:{
+        type: String,
+        required: true,
+    },
+    genre: {
+        type:String
+    },
+    tags:{
+        type: String,
```
Comment (laurawong) [inline backend/models/bookModel.js]: is this supposed to be a list of tags?

File: backend/models/bookModel.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    },
+
+    fav: {
+        type: Boolean,
+        default: true,
```
Comment (laurawong) [inline backend/models/bookModel.js]: it's defaulted to be a favourite?

File: backend/models/quoteModel.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        required: true,
+    },
+    note:{
+        type: String,
+        required: true,
+    },
+    tag:{
+        type: String,
+    },
+    wantToRmb: {
```
Comment (laurawong) [inline backend/models/quoteModel.js]: strange variable name. what does it mean?

File: backend/routes/books.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      .catch(err =>
+        res.status(400).json({ error: 'Unable to update the Database' })
+      );
+  });
+
+router.route('/:id').delete((req,res) => {
+    Book.findByIdAndDelete(req.params.id)
+        .then(() => res.json('Book deleted'))
+        .catch(err => res.status(400).json('Error: ' + err))
+})
```
Comment (laurawong) [inline backend/routes/books.js]: duplicate code?

File: backend/routes/quotes.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+//         quote.graduated = req.body.graduated;
+//         quote.currentInterval = Number(req.body.currentInterval);
+//         quote.whenToShow = Date.parse(req.body.whenToShow);
+
+//         quote.save()
+//             .then(() => res.json('Quote Updated!'))
+//             .catch(err => res.status(400).json('Error: ' + err))
+//     })
+//     .catch(err => err.status(400).json('Error: ' + err))
+// })
```
Comment (laurawong) [inline backend/routes/quotes.js]: bad practice to have dead code

**Review by laurawong — COMMENTED** [review]
not finished reviewing. format files please.

File: backend/models/bookModel.js (hunk 1 above)
Comment (strawberrie68) [reply backend/models/bookModel.js]: yes lol is that bad?

File: backend/models/bookModel.js (hunk 2 above)
Comment (strawberrie68) [reply backend/models/bookModel.js]: should it be defaulted to false?

File: backend/models/quoteModel.js (hunk 3 above)
Comment (strawberrie68) [reply backend/models/quoteModel.js]: should I just use fav?

File: backend/models/bookModel.js (hunk 1 above)
Comment (laurawong) [reply backend/models/bookModel.js]: i'm not that familiar with mongoose, but it doesn't seem like it's a list

File: backend/models/bookModel.js (hunk 2 above)
Comment (laurawong) [reply backend/models/bookModel.js]: in general, booleans are defaulted to false unless there's a reason. why would the user want every book to be auto-favourited?

File: backend/models/quoteModel.js (hunk 3 above)
Comment (laurawong) [reply backend/models/quoteModel.js]: i don't know what it means

File: backend/models/bookModel.js (hunk 2 above)
Comment (strawberrie68) [reply backend/models/bookModel.js]: idk if they just added the quote, doesnt it mean they want to rmb?

**Pull request closed without merging**